tests/test_hello.py

```diff
@@ -2,42 +2,28 @@
 
 
 def test_read_hello(datadir):
-    filename = datadir['hello.txt']
-    with open(filename) as fp:
+    assert set(os.listdir(str(datadir))) == {'hello.txt', 'over.txt'}
+    with open(str(datadir / 'hello.txt')) as fp:
```

Comment:
Unfortunately I couldn't use [read_bytes](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_bytes) or [write_bytes](https://docs.python.org/3/library/pathlib.html#pathlib.Path.write_bytes) on the tests themselves because those functions were added in Python 3.5. :disappointed_relieved: